spec/controllers/availability_zone_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+describe AvailabilityZoneController do
+  describe "#show" do
+    before do
+      EvmSpecHelper.create_guid_miq_server_zone
+      @zone = FactoryGirl.create(:availability_zone)
+      @user = FactoryGirl.create(:user_admin)
+      login_as @user
+    end
+
+    subject do
+      get :show, :id => @zone.id
+    end
+
+    context "render listnav partial" do
+      render_views
+
+      it { is_expected.to have_http_status 200 }
```

Comment:
uf, I merged it already but failed to push it so you might be commenting code that is no longer there, right?